Samples/Salesforce.Sample.SmartSyncExplorer/Salesforce.Sample.SmartSyncExplorer.csproj

```diff
@@ -181,6 +181,9 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <SDKReference Include="SQLite.UAP.2015, Version=3.9.1">
+      <Name>SQLite for Universal App Platform</Name>
+    </SDKReference>
     <SDKReference Include="WindowsDesktop, Version=10.0.10240.0">
```

Comment:
Isnt this the same as Sqlite.WinRT.UAP. The difference is Sqlite.WinRT.UAP is a nuget package